Formatter/FormatterInterface.php

```diff
@@ -16,23 +16,17 @@
 interface FormatterInterface
 {
     /**
-     * @abstract
```

Comment:
@rande Can you say something about the ``@abstract`` annotation? Could I remove this or should it be keept on interfaces?